tools/defaults.bzl

```diff
@@ -87,3 +87,73 @@ def pkg_npm(build_package_json_from_template = False, deps = [], **kwargs):
         deps = deps,
         **kwargs
     )
+
+def generate_ts_proto_module(name, protofile, visibility = ["//visibility:public"]):
```

Comment:
Can we move this into a separate `.bzl` file? I think this is rather a macro than a default at this point (given the amount of code)

Reply:
I initially did this, then second guessed myself :D